shakenfist/model.py

```diff
@@ -0,0 +1,43 @@
+# generated by datamodel-codegen:
```

Comment:
This is obviously generated, is there a thing we need to do if we add a new config option? I feel like we need to document that somewhere because I will forget...

Reply:
Ah, this file snuck in. It was a reference while I wrote the config.py changes.